### ISSUE TITLE
Abstract classes don't work anymore

### ISSUE THREAD

**Issue by RafRaf** [tortoise/tortoise-orm#230]
I can't get extra fields inheriting from an abstract class. Is that feature still available? Example:

```python
from tortoise.models import Model
from tortoise import fields


class AbstractModel(Model):
    new_field = fields.CharField(max_length=100)

    class Meta:
        abstract = True


class ModelOne(AbstractModel):
    id = fields.IntField(pk=True)
    name = fields.TextField()
```

After schema initialization we got only id & name fields in ModelOne:

```
postgres=# \d modelone
                            Table "public.modelone"
 Column |  Type   | Collation | Nullable |               Default
--------+---------+-----------+----------+--------------------------------------
 id     | integer |           | not null | nextval('modelone_id_seq'::regclass)
 name   | text    |           | not null |
Indexes:
    "modelone_pkey" PRIMARY KEY, btree (id)
```
There is no `new_field` here. 

PS: tortoise-orm==0.14.0, asyncpg==0.19.0

**Comment by grigi** [comment]
Hi, we did not explicitly remove the feature.
I suppose that this means our test suite doesn't cover this particular use-case.

What was the last version you used that had this working?

**Comment by RafRaf** [comment]
Hi, @grigi . I've downgraded it to v0.13.4 and it works again.

**Comment by grigi** [comment]
Just for clarity, is this regression since 0.13.5 or just that you have a known good with 0.13.4?

Regardless, I'll try and replicate the issue locally in CI to prevent a future regression of the kind.

**Comment by RafRaf** [comment]
I'm not sure this regression is since 0.13.5. :-(

**Comment by grigi** [comment]
Hi, unfortunately I'll only be able to get to this next week. Sorry for the delay.

**Comment by grigi** [comment]
Ok, this should be fixed. Found the regression, it was during the cleanup after dropping py3.5 support. The tests didn't cover this case :-(
Should be fixed in next release.

**Comment by grigi** [comment]
I just released v0.14.2, please test that it resolved the issue for you :-)

**Comment by RafRaf** [comment]
Well done. 👍 Thank you!!
